### PR TITLE
Update to emoji function

### PR DESCRIPTION
**Pull request by nikchi**
Ability to output emoji sequentially instead of declaring all emoji at compile time

### CODE REVIEW

File: keyboards/m10a/m10a.h [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 #include "quantum.h"
 
-#define LAYOUT( \
+#define M10A( \
```
Comment (drashna) [inline keyboards/m10a/m10a.h]: This should be layout, not M10A.

File: keyboards/m10a/rules.mk [hunk 2]
```diff
@@ -50,19 +50,19 @@ OPT_DEFS += -DBOOTLOADER_SIZE=4096
...
-CONSOLE_ENABLE = no         # Console for debug(+400)
-COMMAND_ENABLE = no        # Commands for debug and configuration
-NKRO_ENABLE = yes            # Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
-BACKLIGHT_ENABLE = yes      # Enable keyboard backlight functionality
-MIDI_ENABLE = no            # MIDI controls
-AUDIO_ENABLE = no           # Audio output on port C6
-UNICODE_ENABLE = yes         # Unicode
-BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
-RGBLIGHT_ENABLE = no        # Enable WS2812 RGB underlight.
+BOOTMAGIC_ENABLE ?= no       # Virtual DIP switch configuration(+1000)
```
Comment (drashna) [inline keyboards/m10a/rules.mk]: There should not have the question mark.  They're not needed.

File: keyboards/m10a/readme.md [hunk 3]
```diff
@@ -1,13 +0,0 @@
-# Rama Works X Machine Industries M10-A Macropad
```
Comment (drashna) [inline keyboards/m10a/readme.md]: The readme files are needed and shouldn't be removed

File: keyboards/m10a/keymaps/gam3cat/keymap.c [hunk 4]
```diff
@@ -1,161 +0,0 @@
-#include "m10a.h"
```
Comment (drashna) [inline keyboards/m10a/keymaps/gam3cat/keymap.c]: This doesn't appear to be your keymap, so please don't remove it

File: keyboards/m10a/keymaps/gam3cat/config.h [hunk 5]
```diff
@@ -1,2 +0,0 @@
-#include "../../config.h"
```
Comment (drashna) [inline keyboards/m10a/keymaps/gam3cat/config.h]: This doesn't appear to be your keymap, so please don't remove it

File: keyboards/m10a/keymaps/gam3cat/readme.md [hunk 6]
```diff
@@ -1,21 +0,0 @@
-# Keymap Maintainer: Gam3cat
```
Comment (drashna) [inline keyboards/m10a/keymaps/gam3cat/readme.md]: This doesn't appear to be your keymap, so please don't remove it

**Comment by nikchi** [comment]
going to repull from clean start

**Pull request closed without merging**